src/commands/test.js

```diff
@@ -1,525 +1,63 @@
+const { Binary } = require('binary-install-raw')
+const { XMLHttpRequest } = require('xmlhttprequest')
```

Comment:
We already have `node-fetch` installed in the project, have you considered using it instead? Not sure if there's a specific reason for using this one.